doc/HD44780.rst

```diff
@@ -0,0 +1,197 @@
+HD44780
+-------
+
+.. image:: images/hd44780_clock.jpg
+
+The HD44780 is a popular LCD display that is widely available and cheap.  It
+is available in several sizes with 16 character x 2 line and  20 character
+by 4 line displays the most common.  As with most LCD displays, they come with
+a backlight which Luma.LCD can control either using PWM modulation or with a
+simple GPIO driven switch.
+
+They are normally connected to SBCs using a parallel ('bit bang') style interface
+but are also sold pre-connected to a few different I2C backpacks including the
+PCF8574, MCP23008 and MCP23017 expanders.
+
+.. Note:
+  Currently luma.lcd only supports the parallel and pcf8574 interfaces
+
+
+Capabilities
+^^^^^^^^^^^^
+The HD44780 is mainly a character based device and can be purchased with different
+font tables installed in its ROM.  The two most popular are the A00 ROM (English/Japanese)
+and the A02 ROM (English/European).  See :download:`HD44780 <tech-spec/HD44780.pdf>`
+for details on which characters are included in each.
+
+In addition to the built-in characters, HD44780s can have up to eight custom
+written into their RAM.  This feature is used by Luma.LCD to provide a limited
```

Comment:
i think we name it using all lowercase, so: luma.lcd

Reply:
There is some small inconsistencies in the documentation on that point but I've adjusted this plus another instance in intro.rst.